service/routes.py

```diff
@@ -47,7 +47,11 @@ def index():
 def list_inventory():
     """Returns all of the inventory"""
     app.logger.info("Request for inventory list")
-    return {}, status.HTTP_200_OK
+    inventory = Inventory.all()
+    results = [inventory_item.serialize() for inventory_item in inventory]
+    app.logger.info(results)
```

Comment:
maybe later on, we can make this debug as we don't want info having the entire list each time API is called

Reply:
👍